src/cpp/subscriber/StatisticsParticipantListener.cpp

```diff
@@ -51,7 +53,8 @@ std::string to_string(
  * @return true when the entity id corresponds to a builtin statistics writer.
  */
 inline bool is_statistics_builtin(
-        const fastrtps::rtps::EntityId_t& entity_id)
+        const fastdds::
+                rtps::EntityId_t& entity_id)
```

Comment:
Apply to remain occurrences
```suggestion
        const fastdds::rtps::EntityId_t& entity_id)
```